src/Defs/Alt.h

```diff
@@ -182,6 +182,7 @@ namespace lua::Class
 		static int WorldToScreen(lua_State* L);
 		static int ScreenToWorld(lua_State* L);
 		static int GetCamPos(lua_State* L);
+		static int GetScreenResolution(lua_State* L)
```

Comment:
do you check your code if its building correctly? seems like its missing a semicolon like the last time lol